docs/installation.md

```diff
@@ -0,0 +1,244 @@
+---
+id: installation
+title: Installation
+---
+
+## CLI
+
+The recommended way to install the scalafmt command line tool is with
+[Coursier](#Coursier).
+
+### Coursier
+
+**NOTE** To install Coursier see [here](https://github.com/coursier/coursier#command-line).
+
+Create a standalone executable in `/usr/local/bin/scalafmt` with (sudo if necessary):
+
+```sh
+coursier bootstrap com.geirsson:scalafmt-cli_2.12:@VERSION@ \
```

Comment:
Right, it would probably make sense to have a `STABLE_VERSION` of some sort here. It can be computed in sbt and injected in the build.